listenbrainz/config.py.sample

```diff
@@ -73,15 +73,29 @@ PLAYING_NOW_MAX_DURATION = 10 * 60
 
 # LOGGING
 
-#LOG_FILE_ENABLED = True
-#LOG_FILE = "./listenbrainz.log"
-
-#LOG_EMAIL_ENABLED = True
-#LOG_EMAIL_TOPIC = "ListenBrainz Webserver Failure"
-#LOG_EMAIL_RECIPIENTS = []  # List of email addresses (strings)
-
-#LOG_SENTRY_ENABLED = True
-#SENTRY_DSN = ""
+# LOG_FILE = {
+#    'filename': './logs/listenbrainz.txt',
+#    'max_bytes': 512 * 1024, # optional
+#    'backup_count': 100, # optional
+# }
+
+# LOG_EMAIL = {
+#    'mail_server': 'localhost',
+#    'mail_port': 25,
+#    'mail_from_host': 'example.org',
+#    'log_email_recipients': [
+#        'user1@example.org',
+#        'user2@example.org',
+#    ],
+#    'log_email_topic': 'ListenBrainz Error',
+#    'level': 'ERROR',
+# }
+
+# LOG_SENTRY = {
+#    'dsn':'',
+#    'environment': 'development',
+# }
+# SENTRY_DSN_PUBLIC = ''
```

Comment:
What is this? is this for sentry reporting from javascript? It doesn't look like it's used anywhere in LB yet. What do we prefer to do? Leave it here for consistency over all projects, or skip it until it's required?